### DISCUSSION TITLE
Using different colour schemes for different charts within the same app

### DISCUSSION THREAD

**Discussion by edkennard** [GSI-CS-CO/chart-fx#496] [Q&A] [answered]
Hi,

I want to apply different colour schemes to different charts in my application, but since DefaultRenderColorScheme is static it seems calls to `DefaultRenderColorScheme.strokeColorProperty().set(chartSpecificColourScheme)` as each chart is instantiated causes issues with charts being rendered with the wrong schemes.

What is the recommended approach to applying per-chart colour schemes?

**Comment by wirew0rm** [comment, marked as answer]
I think that this is not easily possible at the moment. You can set per chart/renderer CSS sheets, but these only affect the axes/grid/etc but not the individual datasets. Ideally we should also aim to support CSS there, but that is not something that will be implemented very quickly on our own. As always, outside contributions for this are very welcome.
At the moment, if you need this level of control, you will have to set the styles individually for each dataset or renderer, i'm afraid.

**Comment by edkennard** [comment]
The use case here was to use different colour schemes for line charts versus bar charts. WIth the line charts we want thin lines with really strong and vibrant colours, but using those same colours in a filled-in bar chart is too intense.

This is not a showstopper though, I'm filling the bar charts via the dataset and it's fine since we don't need too many colours.

Thanks

Ed
